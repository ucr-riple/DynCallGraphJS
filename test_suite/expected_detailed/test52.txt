{
    'global (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test52.js:23:1:23:16)': [
        'main (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test52.js:1:1:22:2)'
    ],
    'main (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test52.js:11:1:11:14)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test52.js:3:16:5:10)'
    ],
    'main (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test52.js:15:5:15:18)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test52.js:12:19:14:2)'
    ],
    'main (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test52.js:15:1:15:21)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test52.js:6:17:8:10)'
    ],
    'main (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test52.js:16:1:16:19)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test52.js:3:16:5:10)'
    ],
    'main (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test52.js:17:1:17:16)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test52.js:6:17:8:10)'
    ],
    'main (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test52.js:20:9:20:19)': [
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test52.js:3:16:5:10)',
        'anon (/Users/madhurimachakraborty/Documents/GitHub/DynCallGraphJS/examples/test52.js:6:17:8:10)'
    ]
}