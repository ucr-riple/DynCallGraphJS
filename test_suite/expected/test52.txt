{ '(test52.js:23:1:23:16)':
   [ '(test52.js:1:1:22:2)' ],
  '(test52.js:11:1:11:14)':
   [ '(test52.js:3:16:5:10)' ],
  '(test52.js:15:5:15:18)':
   [ '(test52.js:12:19:14:2)' ],
  '(test52.js:15:1:15:21)':
   [ '(test52.js:6:17:8:10)' ],
  '(test52.js:16:1:16:19)':
   [ '(test52.js:3:16:5:10)' ],
  '(test52.js:17:1:17:16)':
   [ '(test52.js:6:17:8:10)' ],
  '(test52.js:20:9:20:19)':
   [ '(test52.js:3:16:5:10)',
     '(test52.js:6:17:8:10)' ] }